src/Client.ts

```diff
@@ -135,9 +147,12 @@ export function defaultOptions(opts?: Partial<ClientOptions>): ClientOptions {
 export default class Client {
   address: string
   keys: PrivateKeyBundleV1
+  signedKeys: PrivateKeyBundleV2
```

Comment:
I don't love this, but it feels preferable to doing the `fromLegacyBundle` conversion in every method